README.md

```diff
@@ -1,168 +1,148 @@
-# Polynomials
+# Polynomials.jl
```

Comment:
I should put in the old names here as alternatives and announce plans to deprecate.

Should you write something up about the Chebyshev polynomial types?

Reply:
Yes, I'll add a little list with currently implemented types

Reply:
In regards of the announcements we should put that in the docs, too. A simple admonition like
```md
!!! note
    We are planning to deprecate the `Poly` type [...]
```